### PR TITLE
Pep8 compliance

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: boa/metadata.py [hunk 1]
```diff
@@ -339,8 +332,8 @@ def get_hash_contents(self):
 
         # trim_build_only_deps(self, dependencies)
         dependencies = (
-            self.get_dependencies("build")
-            + self.get_dependencies("host")
+            self.get_dependencies("build") +
+            self.get_dependencies("host")
             # self.output.requirements["build"] + self.output.requirements["host"]
         )
         dependencies = {x.name for x in dependencies}
```
Comment (martinRenou) [inline boa/metadata.py:339]: Line https://github.com/mamba-org/boa/pull/42/files#diff-2e1b4be18dab7f82649cc72e80bcb0f1d9923bef8be4afc036600785e7e8d3feR356

It complains with `W605 invalid escape sequence '\s'`, any idea why?

File: boa/cli/mambabuild.py [hunk 2]
```diff
@@ -78,7 +78,6 @@ def __init__(self, channels, platform):
         self.repos = []
```
Comment (martinRenou) [inline boa/cli/mambabuild.py:78]: Line https://github.com/mamba-org/boa/pull/42/files#diff-6d845a6fe42e99697ca13ec691bcc8aa9b5e053dd3a837f53f3689209c734b6fR41

It complains with `F821 undefined name 'installed_pkg_recs'`, I don't know how to fix it :S

File: boa/cli/boa.py [hunk 3]
```diff
@@ -793,15 +789,15 @@ def main(config=None):
...
-    convert_parser = subparsers.add_parser(
+    subparsers.add_parser(
         "convert",
         parents=[parent_parser],
         help="convert recipe.yaml to old-style meta.yaml",
     )
-    build_parser = subparsers.add_parser(
+    subparsers.add_parser(
         "build", parents=[parent_parser], help="build a recipe"
     )
```
Comment (martinRenou) [inline boa/cli/boa.py:803]: Line https://github.com/mamba-org/boa/pull/42/files#diff-861ef9dd4629a87f900b32efabb8d0c7880b87d61ded76ce22b4541344f0a401R952

It complains with `E722 do not use bare 'except'`, I don't know what kind of Exception is supposed to be raised so again I am not sure how to fix it

**Comment by wolfv** [comment]
this is great! 
I'll install pre-commit and go over the remaining issues

**Pull request merged**